assets/sass/components/sign-in-with-google/_googlesitekit-sign-in-with-google-settings-module.scss

```diff
@@ -97,6 +99,10 @@
 				.mdc-layout-grid {
 					.mdc-layout-grid__inner {
 						margin-bottom: $grid-gap-desktop;
+
+						@media (min-width: $bp-xlarge) {
```

Comment:
Looks like IB is incorrect. This change doesn't do anything. Instead, we need to adjust the `justify-content` setting that adds the excessive space between the client ID and the button style rows. After changing this, we should have VRT images changed as well.